README.md

```diff
@@ -1,31 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
```

Comment:
- [ ] **[OPTIONAL]** I would suggest giving your project an appropriate name. **'Project Name'** is merely a dummy text and so is too generic to be your project name